README.rst

```diff
@@ -1 +1,5 @@
 This is the Artefactual Storage Service for Archivematica.
+
+Start worker:
+
+celery -A storage_service worker -l info
```

Comment:
Not yet.  Updating the archivematica ansible role is going to be addressed in this ticket:
https://projects.artefactual.com/issues/10285, which is assigned to hakamine.  

That ticket will blocks merging this PR, I think.